src/github.com/getlantern/errors/errors.go

```diff
@@ -308,7 +325,14 @@ func buildError(desc string, wrapped error, cause Error) *structured {
 			}
 		}
 	}
-	e.data["error"] = hidden.Clean(desc)
+
+	cleanedDesc := hidden.Clean(desc)
+	e.data["error"] = cleanedDesc
```

Comment:
Note that error now contains the description without format variables.  The fully rendered description is stored in error_text.  Of course this only works when using `errors.New`, not `fmt.Errorf`, so we should get in the habit of using `errors.New` wherever possible.